contributing/contributing_to_datacommons.md

```diff
@@ -21,18 +29,42 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
-## Code reviews
+### Becoming familiar with the project
+
+You can find suggested reading about Data Commons concepts in [the Background section](/contributing/background/background.html).
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
+All submissions, including submissions by project members, require review. Data Commons
+uses GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
-## Style guide
+This project follows Google's [code style guide](https://google.github.io/styleguide/), [documentation style guide](https://developers.google.com/style), and [open source community guidelines](https://opensource.google/conduct/).
+
+## Your first contribution
+
+### Contributing a dataset
+
+Directions are available on [the 'Adding Datasets' page](/contributing/adding_datasets.html).
+
+### Creating a new tool
+
+Data Commons welcomes the development of new tools that make the data on Data Commons more accessible to end users. If you would like to propose a new tool, feel free to create a PR in [the Data Commons tools repo](https://github.com/datacommonsorg/tools) or contact the team at <support@datacommons.org>.
+
+### Sharing analysis
+
+Data Commons is currently posting example analyses on [the tutorials page](/tutorials.html). To submit your analysis as a tutorial, open a PR in [the Data Commons documentation website repo](https://github.com/datacommonsorg/docsite/pulls) with the title of the analysis and links to its Colab notebook and Github raw .ipynb file. Your addition to the tutorial page should look like this when raw:
+
+```
+-   ** <YOUR TITLE HERE> **
+
+    [Google Colab](*YOUR COLAB LINK HERE*){:target="_blank"}, [Github](*YOUR GITHUB .IPYNB FILE HERE){:target="_blank"}
```

Comment:
I wonder if we should also use <> for the other placeholders, since it's the same block.

Reply:
fixed!